Projects/l0/Sniffer/lib/Sniffer/sniffer_com.h

```diff
@@ -0,0 +1,62 @@
+/******************************************************************************
+ * @file sniffer_com
+ * @brief communication driver
+ * @author Luos
+ * @version 0.0.0
+ ******************************************************************************/
+#ifndef SNIFFER_COM_H
+#define SNIFFER_COM_H
+
+#include "cmd.h"
+#include "sniffer_data.h"
+#include "streaming.h"
```

Comment:
I don't think you need any of those here.
Watch out if a .h is only needed into a .c file, you should include it on the .c. It's allowing to keep it private.